kolibri/content/recommendations.py

```diff
@@ -0,0 +1,32 @@
+from collections import Counter
+
+# from kolibri.logger.models import ContentInteractionLog
+from kolibri.content import api
+from kolibri.content.models import ContentNode
+
+def recommendations_interaction_log():
+    """
+    Calculates the content that is accessed the most and returns a subset of that.
+
+    :return: Queryset of the the most frequently accessed content
+    """
+    # logs = ContentInteractionLog.objects.filter(channel_id=channel_id).all()
+    logs = ContentNode.objects.all()
+    content_count = Counter()
+    for log in logs:
+        content_count[log.content_id] += 1
```

Comment:
```
from django.db.models import Count
content_count_sorted = ContentInteractionLog.objects.filter(channel_id=channel_id).values('content_id').annotate(num_content=Count('content_id').order_by('-num_content')
```

I think this is what you are looking for - can do the counting and sorting in what I think is a single query.